src/test/java/kuit/subway/station/StationTest.java

```diff
@@ -0,0 +1,81 @@
+package kuit.subway.station;
+
+import io.restassured.RestAssured;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import kuit.subway.AcceptanceTest;
+import net.minidev.json.JSONObject;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+public class StationTest extends AcceptanceTest {
+    @Test
+    void createStation() {
+        JSONObject jsonObj = new JSONObject();
+        jsonObj.put("name", "강남역");
```

Comment:
보통 JSONObject를 바로 쓰진 않고, Map에 데이터를 넣는 방식을 자주 사용해요.

```suggestion
        Map<String, String> body = new HashMap<>();
        body.put("name", "강남역");
```